projects/core/src/model/configurator.model.ts

```diff
@@ -96,6 +96,13 @@ export namespace Configurator {
     configId: string;
     ownerKey: string;
   }
+
+  export interface ReadFromCartEntryParameters {
```

Comment:
Why is it called ReadFromCartEntryParameters?
Would ReadCartEntryParameters not be better?

Reply:
Renamed to ReadConfigurationFromCartEntryParameters